client-data/board.html

```diff
@@ -20,6 +20,7 @@
 	{{#languages}}
 	<link rel="alternate" hreflang="{{.}}" href="{{../boardUriComponent}}?lang={{.}}" />
 	{{/languages}}
+	<script src="https://cdn.jsdelivr.net/npm/promise-polyfill@8/dist/polyfill.min.js"></script>
```

Comment:
Currently, WBO doesn't have any dependency to external services. This is good for privacy, and ease of deployment behind a firewall. Maybe we can use something like [polyfill-library](https://www.npmjs.com/package/polyfill-library) on the server instead ? This would also have the advantage of providing many other polyfills, to let us use more modern js features in the code, and ensure compatibility with many old browsers.

Reply:
That would probably be the way to go, do you want to go ahead and change it?

Reply:
Yes, I can do that :)

Reply:
I have gone ahead and implemented it